src/test/crimson/seastore/test_transaction_manager.cc

```diff
@@ -899,15 +901,40 @@ TEST_F(transaction_manager_test_t, random_writes_concurrent)
       [&](auto) {
 	return seastar::async([&] {
 	  while (writes < 300) {
-	    auto t = create_transaction();
-	    auto ext = get_extent(
```

Comment:
Maybe better to implement `try_get_extent(...)` to handle eagain and return whether it is successful.
Then increment `failures` if not successful, no need to `repeat_eagain`.

I think this is the original intention.

Reply:
updated.